packages/@guardian/source-react-components-development-kitchen/components/alert-switch/AlertSwitch.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import { AlertSwitch, AlertSwitchProps } from './AlertSwitch';
+import type { Story } from '../../../../../lib/@types/storybook-emotion-10-fixes';
+import {
+	asPlayground,
+	asChromaticStory,
+} from '../../../../../lib/story-intents';
+import { useState } from 'react';
+import { ArticlePillar, ArticleSpecial } from '@guardian/libs';
+import { css } from '@emotion/react';
+
+export default {
+	title: 'Kitchen/source-react-components-development-kitchen/Alert Switch',
```

Comment:
We've mostly used TitleCase without spaces for the stories names (although we haven't done this everywhere 🙁 )

```suggestion
	title: 'Kitchen/source-react-components-development-kitchen/AlertSwitch',
```